app/assets/javascripts/admin/products/controllers/units_controller.js.coffee

```diff
@@ -1,3 +1,4 @@
+# Controller for New Products form
 angular.module("admin.products")
```

Comment:
:question: Oh, I see. I first thought that this is the _new_ BUU products form. But you mean the form for "new products". Maybe "Controller for form to create new products"?

Well, it will hopefully be gone soon anyway...

Reply:
🤦 Haha sorry! The capitalisation matters ;)